src/main/java/org/toradocu/translator/ConditionTranslator.java

```diff
@@ -443,6 +444,8 @@ private static void processTag(Tag tag, DocumentedMethod method) {
                     "Shouldn't be",
                     ". " + ((ParamTag) tag).parameter().getName() + " shouldn't be");
       }
+
+      if (tag.getKind() == Tag.Kind.RETURN) {}
```

Comment:
Yes, it is. We have to find the patterns for this kind of tag yet, so, it is just a placeholder for the incoming treatment of those tags.